static/css/inline-selection.scss

```diff
@@ -117,165 +42,137 @@ $overlay-z-index: 9999999999;
   border: 0;
 }
 
+.hover-highlight {
+  animation: fade-in 125ms forwards $bezier;
+  background: rgba(255, 255, 255, 0.1);
+  border-radius: 1px;
+  pointer-events: none;
+  position: absolute;
+  z-index: $overlay-z-index + 1;
+}
+
 .mover-target.direction-topLeft {
   cursor: nwse-resize;
-  top: -$mover-outer;
+  height: $mover-size;
   left: -$mover-outer;
+  top: -$mover-outer;
   width: $mover-size;
-  height: $mover-size;
 }
 
 .mover-target.direction-top {
   cursor: ns-resize;
-  top: -$mover-outer;
+  height: $mover-size;
   left: 0;
+  top: -$mover-outer;
   width: 100%;
-  height: $mover-size;
   // It will be too wide, but this will place it below topLeft/etc:
   z-index: 4;
 }
 
 .mover-target.direction-topRight {
   cursor: nesw-resize;
-  top: -$mover-outer;
-  right: -$mover-outer;
   height: $mover-size;
+  right: -$mover-outer;
+  top: -$mover-outer;
   width: $mover-size;
 }
 
 .mover-target.direction-left {
   cursor: ew-resize;
-  top: 0;
-  left: -$mover-outer;
   height: 100%;
+  left: -$mover-outer;
+  top: 0;
   width: $mover-size;
   z-index: 4;
 }
 
 .mover-target.direction-right {
   cursor: ew-resize;
-  top: 0;
-  right: -$mover-outer;
   height: 100%;
+  right: -$mover-outer;
+  top: 0;
   width: $mover-size;
   z-index: 4;
 }
 
 .mover-target.direction-bottomLeft {
+  bottom: -$mover-outer;
   cursor: nesw-resize;
+  height: $mover-size;
   left: -$mover-outer;
-  bottom: -$mover-outer;
   width: $mover-size;
-  height: $mover-size;
 }
 
 .mover-target.direction-bottom {
+  bottom: -$mover-outer;
   cursor: ns-resize;
+  height: $mover-size;
   left: 0;
-  bottom: -$mover-outer;
   width: 100%;
-  height: $mover-size;
   z-index: 4;
 }
 
 .mover-target.direction-bottomRight {
-  cursor: nwse-resize;
   bottom: -$mover-outer;
+  cursor: nwse-resize;
+  height: $mover-size;
   right: -$mover-outer;
   width: $mover-size;
-  height: $mover-size;
 }
 
 .mover-target:hover .mover {
-  opacity: 1;
+  transform: scale(1.05);
 }
 
 .mover {
-  background-color: $link-blue;
-  border-radius: 2px;
-  box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.1) inset;
+  background-color: $white;
+  border-radius: 50%;
+  box-shadow: 0 0 4px rgba(0, 0, 0, 0.5);
   height: $handle-size;
-  opacity: 0.5;
-  position: absolute;
-  transition: opacity 150ms, width 150ms, height 150ms;
+  opacity: 1;
+  position: relative;
+  transition: transform 125ms $bezier;
   width: $handle-size;
-  z-index: 1001;
 
   .small-selection & {
     height: $handle-size-small;
     width: $handle-size-small;
   }
 }
 
-.direction-topLeft .mover {
-  top: $handle-mover-offset;
+.direction-topLeft .mover,
+.direction-left .mover,
+.direction-bottomLeft .mover {
```

Comment:
Oh, huh... maybe the handles are now being done with flex, and that's how these rules get folded together?  I guess this could be okay, but it makes me worried.